lib/util.js

```diff
@@ -1,15 +1,22 @@
 const util = require('util');
-const fs = require('fs');
+const path = require('path');
+const which = require('which');
+const { pathExists, readFile } = require('fs-extra');
 
 const childProcess = require('child_process');
 
 const { promisify } = util;
 const exec = promisify(childProcess.exec);
-const readFile = promisify(fs.readFile);
 const wait = promisify(setTimeout);
 
+async function packageExist(pckg, dir) {
+  return pathExists(path.resolve(dir, `node_modules/.bin/${pckg}`));
+}
+
 module.exports = {
   exec,
   readFile,
   wait,
+  packageExist,
+  which: promisify(which),
```

Comment:
Why promisify it if we are using the `sync` version everywhere?

Reply:
I originally used `async` version but switched to sync later. Forgot to change it. Good catch!